BookPlayer/Library/Views/BookCellView.swift

```diff
@@ -47,6 +47,9 @@ class BookCellView: UITableViewCell {
             } else if ratio < 1 {
                 self.artworkHeight.constant = 50.0
                 self.artworkWidth.constant = 50.0 * ratio
+            } else {
```

Comment:
I think we can replace this `if ... else if ... else` for these two lines:
```
self.artworkHeight.constant = ratio > 1 ? 50.0 / ratio ? 50
self.artworkWidth.constant = ratio < 1 ? 50.0 * ratio ? 50
```

Reply:
Agreed.